packages/kuma-gui/src/test-support/mocks/src/meshes/_/dataplanes/_overview.ts

```diff
@@ -23,20 +25,23 @@ export default ({ fake, pager, env }: EndpointDependencies): MockResponder => (r
       .map(item => { const [key, ...rest] = item.split(':'); return [key, rest.join(':')] }))
     : {}
 
-  let filterType: 'gateway_builtin' | 'gateway_delegated' | 'proxy' | undefined
-  if (_gateway === 'builtin' || _gateway === 'delegated') {
-    filterType = `gateway_${_gateway}`
-  } else if (_gateway === 'false') {
-    filterType = 'proxy'
-  } else if (tags['kuma.io/service']) {
-    if (tags['kuma.io/service'].includes('gateway_builtin')) {
-      filterType = 'gateway_builtin'
-    } else if (tags['kuma.io/service'].includes('gateway_delegated')) {
-      filterType = 'gateway_delegated'
-    } else {
-      filterType = 'proxy'
+  const filterType = (() => {
+    switch (true) {
+      case _gateway === 'false':
+        return 'STANDARD'
+      case defaultType === 'builtin':
+      case _gateway === 'builtin':
+        return 'BUILTIN'
+      case defaultType === 'delegated':
+      case _gateway === 'delegated':
+      case tags['kuma.io/service']?.includes('-delegated'):
+        return 'DELEGATED'
+      case tags['kuma.io/service']?.includes('-builtin'):
+        return 'BUILTIN'
+      default:
+        return ''
```

Comment:
Just reordering a bit to have one return of `BUILTIN` 🙂 

```suggestion
  const filterType = (() => {
    switch (true) {
      case _gateway === 'false':
        return 'STANDARD'
      case defaultType === 'builtin':
      case _gateway === 'builtin':
      case tags['kuma.io/service']?.includes('-builtin'):
        return 'BUILTIN'
      case defaultType === 'delegated':
      case _gateway === 'delegated':
      case tags['kuma.io/service']?.includes('-delegated'):
        return 'DELEGATED'
      default:
        return ''
```

Reply:
The case `_gateway === 'false'` used to return `'proxy'`. Just want to verify that we are going with `'STANDARD'` now?

Reply:
> Just reordering a bit to have one return of BUILTIN 🙂

This means the tags containing `-builtin` is more important than the actual type being `DELEGATED`. This was the reason why it's in the order that it is. I don't think its massively important, but I wanted to keep the previous functionality (which also had this priority/ordering)

> The case _gateway === 'false' used to return 'proxy'. Just want to verify that we are going with 'STANDARD' now?

Yep, going with `STANDARD` so its more similar to `DELEGATED` and `BUILTIN`, from what I remember we don't actually print out `STANDARD` anywhere we just follow the existing pattern of uppercasing. i.e. both the string `proxy` and `STANDARD` aren't anything in the backend (whereas `DELEGATED` and `BUILTIN` are).

Reply:
> > Just reordering a bit to have one return of BUILTIN 🙂
> 
> This means the tags containing `-builtin` is more important than the actual type being `DELEGATED`. This was the reason why it's in the order that it is. I don't think its massively important, but I wanted to keep the previous functionality (which also had this priority/ordering)

Ah, gotcha 😄 nvm then

> > The case _gateway === 'false' used to return 'proxy'. Just want to verify that we are going with 'STANDARD' now?
> 
> Yep, going with `STANDARD` so its more similar to `DELEGATED` and `BUILTIN`, from what I remember we don't actually print out `STANDARD` anywhere we just follow the existing pattern of uppercasing. i.e. both the string `proxy` and `STANDARD` aren't anything in the backend (whereas `DELEGATED` and `BUILTIN` are).

Alright, thanks for the explanation 👍